src/libraries/Microsoft.Extensions.Logging.Abstractions/ref/Microsoft.Extensions.Logging.Abstractions.csproj

```diff
@@ -5,5 +5,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Microsoft.Extensions.Logging.Abstractions.cs" />
+    <Compile Include="$(RepoRoot)src\libraries\System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs" />
```

Comment:
Ah sorry @hrrrrustic seems `<Nullable>enable</Nullable>` above was enough, seems after that added the `NullableAttributes` getting added automatically, please remove this line
```suggestion
```